docs/packages.toml

```
@@ -151,6 +151,10 @@
     user = "MadNLP"
     rev = "389561354a774441cd5e6b3aa5fffee102ed222e"
     has_html = true
+[Manopt]
+    user = "JuliaManifolds"
+    rev = "v0.4.42"
+    filename = "Readme.me"
```

```suggestion
    filename = "Readme.md"
```

Sorry my bad (do not comment on PRs before first coffee) – I had a typo in md...